README.md

```diff
@@ -1,5 +1,69 @@
 # phone-history-store-frontend
-
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e1c7c0497f5a487d90610ab41e9db55f)](https://app.codacy.com/gh/nezago/phone-history-store-frontend?utm_source=github.com&utm_medium=referral&utm_content=nezago/phone-history-store-frontend&utm_campaign=Badge_Grade_Dashboard)  [![Maintainability](https://api.codeclimate.com/v1/badges/e5ff93644af293ab0e46/maintainability)](https://codeclimate.com/github/nezago/phone-history-store-frontend/maintainability)  ![Node.js CI](https://github.com/nezago/phone-history-store-frontend/workflows/Node.js%20CI/badge.svg)
 
-This is the official repository of Phone History Store
+## Project vision
+Since the rise of technology has become one of the essential daily life dependency, mobile devices are one of the tools mostly used to facilitate access to the technology. Everyday we make calls to our family members, our friends and our mostly cherished lovers, we talk through phone calls, through short message service (also known as SMS).
+
+There are sometimes we find ourselves in a need of recalling our sweet memories of what we talked with our friends in the past. Hence our mobile phones, or tablets don't have enough space to save the logs, and SMS for us for a long time like 1, 2, 3 or even 10 years.
+
+Here Phone History Store helps in that situation, you can store your sms, your call logs, and you phone book as well, for any desired time. It helps you manage and save securely all your phone history, and you can get your phone history wherever you are, even if you lost your device, but you history will remain with you.
+
+## Project description
+This project *Phone History Store Frontend* is the User Interface of [the backend project](https://github.com/nezago/phone-history-store-backend).
+
+It sync the calls, and SMS to a remote store, and whenever a user needs them, he/she can get them wherever he/she wants them, either on his telephone, or tablet, or computer.
+
+The main logic is that whenever you make, receive, or miss a new call this app will automaticall take the log and saves it on your local store and in a matter of time if you get connected to the internet, it will just send the new record to your remote store. 
+
+## Built with
+1.  [React native](https://reactnative.dev/)
+1.  [SASS](https://sass-lang.com/)
+1.  [Java](https://www.java.com/en/)
+
+## Tools
+1.  [VSCode](https://code.visualstudio.com/)
+1.  [Git](https://git-scm.com/)
+1.  [GitHub](https://www.github.com/)
```

Comment:
Codacy found an issue: [[ordered-list-marker-value] Marker should be `3`, was `1`](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=5868903)